html/browsers/history/the-location-interface/location_hash.html

```diff
@@ -10,6 +10,9 @@
     <iframe id="srcdoc-iframe"
        srcdoc="<div style='height: 200vh'></div><div id='test'></div>"></iframe>
     <script>
+//**This test assumes that assignments to location.hash will be synchronous - this is how all browsers implement it.
+//The spec (as of 25 March 2011) disagrees.
```

Comment:
If this is true, we should track this with an issue against whatwg/html.

Reply:
FWIW, I didn't personally test/verify this statement beyond seeing that all major browsers seem to do this, but this comment is in 001.html and 002.html from the history interface tests:
https://github.com/web-platform-tests/wpt/search?q=%22This+test+assumes+that+assignments+to+location.hash+will+be+synchronous%22&unscoped_q=%22This+test+assumes+that+assignments+to+location.hash+will+be+synchronous%22

This test didn't seem to belong in the history interface though - so I copied it here. I can open a spec issue.

Reply:
As far as I can tell the specification also does synchronous navigation for fragment changes though.

Reply:
Those are very old tests, so probably best not to copy-and-paste information from them.

Reply:
Okay, I've removed the comment in the latest commit. I can also of course remove it from the other tests if you think it would be good to clean up.